### PR TITLE
fix(security): Allocate stage gate port for security-proxy-setup--nginx dependency

### PR DESCRIPTION
**Pull request by bnevis-i**
To fix CORS problem in EdgeX 3.0, we need to block NGINX startup on security-proxy-setup finishing.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [X] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [X] I have fully tested (add details below) this the new feature or bug fix (if not, why?)
- [X] I have opened a PR for the related docs change (if not, why?)
  <[link to docs PR](https://github.com/edgexfoundry/edgex-docs/pull/1188)>

## Testing Instructions
This fix is related to https://github.com/edgexfoundry/edgex-go/pull/4638

### CODE REVIEW

File: compose-builder/common-sec-stage-gate.env [hunk 1]
```diff
@@ -31,3 +31,4 @@ STAGEGATE_REGISTRY_PORT=8500
 STAGEGATE_REGISTRY_READYPORT=54324
 STAGEGATE_READY_TORUNPORT=54329
 PROXY_SETUP_HOST=edgex-security-proxy-setup
+STAGEGATE_PROXYSETUP_READYPORT=54325
```
Comment (lenny-intel) [inline compose-builder/common-sec-stage-gate.env:34]: Need to run `make build` so the compose files get this...

**Review by lenny-intel — CHANGES_REQUESTED** [review]

File: compose-builder/common-sec-stage-gate.env (hunk 1 above)
Comment (bnevis-i) [reply compose-builder/common-sec-stage-gate.env:34]: Thanks.

**Review by lenny-intel — APPROVED** [review]
LGTM

**Review by jim-wang-intel — APPROVED** [review]

**Comment by lenny-intel** [comment]
@cherrycl , unrelated failures on this PR also. THX!

**Comment by cherrycl** [comment]
Jenkins got incorrect SHA1 value, please rebase the PR.

**Pull request merged**